whatwg.org/style/subpages.css

```diff
@@ -74,6 +74,12 @@ dl > dt { margin: 1em 0 0 0; }
 dl > dd { margin: 0.25em 0 0.25em 2em; }
 ol.dated { list-style: disc; }
 
+dl.compact > div { margin: 1em 0; }
+dl.compact > div > dt { display: inline; margin: 0; font: inherit; font-weight: bold; }
+dl.compact > div > dt::after { content: ": "; }
+dl.compact > div > dd { display: inline; margin: 0; }
+dl.compact > div > dd:not(:last-of-type)::after { content: "; "; }
```

Comment:
Does this mean we need to bump `cacheKeyToUse` in https://resources.whatwg.org/website-service-worker.js?